config/environment.rb

```
@@ -0,0 +1,13 @@
+ENV['RACK_ENV'] ||= 'production'
```

I'd opt to make this fail if no environment is defined. Failing fast is more handy in this case than a semi-sane default.

---

@jalcine hmm.. my thinking was to load this file in my `Rakefile` so that I could have a task to seed the DB without having to connect again.

I don't think `RACK_ENV` is set for command line?